modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/impl/SchedulerServiceImpl.java

```diff
@@ -791,6 +803,17 @@ public Boolean apply(MediaPackage mp) {
     }
   }
 
+  private boolean isAdmin() {
```

Comment:
Hmm... I think that the chance that you find this logic somewhere else in the Opencast codebase (-> code re-use)  is very high. Can you please check?

Reply:
So this kind of function does exist in _modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/AssetManagerWithSecurity.java_ [192]

And a similar call is made in 11 files in the project, so I'm thinking that maybe it would be better if this was a public function in the "user class" (_modules/common/src/main/java/org/opencastproject/security/api/User.java_) ref: MH-13470

Reply:
Ok, thanks for checking! 

While the class User would be a better location for this logic, this seems somewhat out-of-scope for this pull request.

I therefore suggest to leave the logic here.

If you have some more time, though, you could simplify this into:

return (securityService.getUser().hasRole(GLOBAL_ADMIN_ROLE)
     || securityService.getUser().hasRole(securityService.getOrganization().getAdminRole());